jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/DataSetDefsBootstrap.java

```diff
@@ -61,13 +61,19 @@ protected void registerDataSetDefinitions() {
                             "log.correlationKey, " +
                             "log.externalId, " +
                             "log.processInstanceDescription, " +
-                            "COALESCE(info.lastModificationDate,log.end_date) as lastModificationDate " +
-                        "from " +
-                            "ProcessInstanceLog log " +
-                        "left join " +
-                            "ProcessInstanceInfo info " +
-                        "on " +
-                            "info.InstanceId=log.processInstanceId"
+                            "COALESCE(info.lastModificationDate,log.end_date) as lastModificationDate, " +
+                            "COALESCE(" +
```

Comment:
Sounds like a candidate to be put to upstream DB tests. We have a whole bunch of these and, this being in backed, I don't see the need for Selenium.